lib/ansible/modules/storage/netapp/na_ontap_volume.py

```diff
@@ -457,7 +470,8 @@ def get_volume(self, vol_name=None):
 
             }
             if volume_space_attributes.get_child_by_name('percentage-snapshot-reserve'):
-                return_value['percent_snapshot_space'] = volume_space_attributes['percentage-snapshot-reserve']
+                vol_perct_snap = volume_space_attributes['percentage-snapshot-reserve']
```

Comment:
I don't see the point of adding these intermediate variables.

Reply:
Apols, I was using python standard formatter that recommends line length of 100 instead of 160 that recommends ansible. I will revert

Reply:
idem, to avoid 100 chars line length, reverting in my next commit